.github/workflows/template_automerge_dependabot.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     name: auto-merge
     runs-on: ubuntu-22.04
 
-    if: github.actor == 'dependabot[bot]'
+    if: github.event.pull_request.user.login == 'dependabot[bot]'
```

Comment:
Or do yo want to use both checks as an OR check

Reply:
For the auto-merging feature, only the creator of the pull request (PR) is important. We do not want to prevent merging when a user rebases the PR (this changes the actor). So, for me, this looks good and sufficient.